src/styles/blocks/header.scss

```
@@ -0,0 +1,32 @@
+.header {
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  &__logo {
+    width: $icon-size;
+    height: $icon-size;
+    left: 50px;
+    top: 10px;
```

```suggestion
     display: block;
    width: $icon-size;
    height: $icon-size;
```
it should fix little mismatch in tests